README.md

```diff
@@ -74,6 +74,32 @@ To use React UI in your app:
    );
    ```
 
+### Development mode
```

Comment:
I think this section better fits the existing _Contributing_ section. So it would end up like this:

- …
- Contributing
  - Package Linking
  - **Development Mode**

Reply:
This has nothing to do with development of React UI, and everything to do with development with React UI. It is therefore relevant to the consumers and not to the contributors.

Thus I think it is placed correctly.

Reply:
Yes, I realised it a while after I posted my comment. Agree.